File: prac1_Python/EJ1/calcula_nota.py
# -*- coding: utf-8 -*-
#Ejercicio 1 => calcula_nota
def calcula_nota(nota):
    if nota<5:
        print(str("El alumno está suspenso."))
    elif nota==5 or nota==6:
        print(str("El alumno está aprobado."))
    elif nota==7 or nota==8:
        print(str("El alumno tiene un notable."))
    elif nota==9 or nota==10:
        print(str("El alumno tiene un sobresaliente."))
    else:
        print(str("Introduzca una nota valida."))

nota = int(input("Introduzca un número: "))
calcula_nota(nota)

File: Practicas_C/prac4_BrunoFrescoOtero/EJ1/calcula_Nota.c
#include <stdio.h>
void calc_Nota(int nota);
int main(){
	int nota_Eva;
	printf("Introduzca la nota a evaluar: ");
	scanf("%d", &nota_Eva);
	calc_Nota(nota_Eva);
	return 0;
}

void calc_Nota(int nota){
	switch(nota){
		case 5:
		case 6: printf("El alumno tiene un aprobado.\n");
			break;
		case 7:
		case 8: printf("El alumno tiene un notable.\n");
			break;
		case 9:
		case 10: printf("El alumno tiene un sobresaliente.\n");
			 break;
		default: printf("EL alumno tiene un suspenso.\n");
	}
	return;
}
